sdks/python/apache_beam/runners/worker/sdk_worker_main.py

```diff
@@ -74,7 +74,7 @@ def do_GET(self):  # pylint: disable=invalid-name
         self.end_headers()
 
         for line in StatusServer.get_thread_dump():
-          self.wfile.write(line)
+          self.wfile.write(str.encode(line, 'UTF-8'))
```

Comment:
Thanks a lot, @angoenka ! A few issues:
1. Please file a JIRA and mark it as 2.16 blocker, it should be an easy fix, however it's an important issue for debuggability of Python streaming. I think it is reasonable to fix it ASAP. cc: @markflyhigh 

2. We can simplify this to `self.wfile.write(line.encode('utf-8'))`

3. Please test that this codepath changes work as expected on Python 3.5 as well as 3.6. Per https://docs.python.org/3/library/http.server.html, there were changes in Python 3.6 affecting wfile implementation, so let's make sure that before Python 3.6 we can still write bytes to this stream.

Reply:
1. Done
2. Done
3. checked with 3.5
4. TODO